projects/aca-content/src/lib/components/details/details.component.ts

```diff
@@ -110,20 +110,6 @@ export class DetailsComponent extends PageComponent implements OnInit, OnDestroy
       .subscribe((aspectActions) => {
         this.aspectActions = aspectActions;
       });
-    this.infoDrawerOpened$
-      .pipe(
-        first((opened) => !opened),
-        takeUntil(
-          merge(
-            this.onDestroy$,
```

Comment:
onDestroy$ can be removed as now it's unused

Reply:
removed, ty